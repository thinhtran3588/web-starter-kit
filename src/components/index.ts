export * from './AdminLayout';
export * from './AdminFooter';
export * from './Footer';
export * from './Header';
export * from './Icon';
export * from './Layout';
export * from './Sidebar';
export * from './SidebarMenu';
export * from './FormHeader';
export * from './Field';
export * from './FormFilter';
export * from './FormInput';
export * from './FormSearch';
export * from './Table';
export * from './LanguageSelection';
export * from './Link';
export * from './AuthLayout';
export * from './Head';
export * from './Button';
export * from './Grid';
export * from './Typography';
