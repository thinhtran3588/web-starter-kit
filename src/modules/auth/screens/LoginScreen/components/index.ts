export * from './EmailLogin';
export * from './PhoneNoLogin';
