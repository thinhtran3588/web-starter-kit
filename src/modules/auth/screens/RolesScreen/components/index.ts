export * from './Detail';
export * from './PermissionsTable';
