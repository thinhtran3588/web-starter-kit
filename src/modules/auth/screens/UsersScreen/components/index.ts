export * from './Detail';
export * from './PermissionsTable';
export * from './ChangePassword';
