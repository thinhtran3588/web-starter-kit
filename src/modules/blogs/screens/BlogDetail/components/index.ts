export * from './Content';
export * from './SideMenu';
