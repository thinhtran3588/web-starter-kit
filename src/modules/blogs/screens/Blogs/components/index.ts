export * from './Categories';
export * from './BlogsList';
