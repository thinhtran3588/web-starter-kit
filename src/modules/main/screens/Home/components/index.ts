export * from './CoverImage';
export * from './Features';
export * from './Quote';
export * from './Help';
