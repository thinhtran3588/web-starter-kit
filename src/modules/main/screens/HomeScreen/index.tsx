import React from 'react';
import { Layout } from '@app/components';
import { WithTranslation, withTranslation } from '@app/core';
import { CoverImage, Features, Quote, Help } from './components';

type Props = WithTranslation;

const Screen = (props: Props): JSX.Element => {
  const { t } = props;
  return (
    <Layout title={t('home')} description={t('description')}>
      <CoverImage t={t} />
      <Features t={t} />
      <Quote t={t} />
      <Help t={t} />
    </Layout>
  );
};

Screen.getInitialProps = async () => {
  return {
    namespacesRequired: ['common', 'home'],
  };
};

export const HomeScreen = withTranslation('home')(Screen);
