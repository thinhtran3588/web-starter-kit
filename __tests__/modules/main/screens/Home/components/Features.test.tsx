import React from 'react';
import { render } from '@testing-library/react';
import { Features } from '@app/modules/main/screens/HomeScreen/components';

describe('@app/modules/main/screens/HomeScreen/components/Features', () => {
  beforeEach(() => {});

  it('renders successfully', async () => {
    const { baseElement } = render(<Features />);
    expect(baseElement).toMatchSnapshot();
  });
});
