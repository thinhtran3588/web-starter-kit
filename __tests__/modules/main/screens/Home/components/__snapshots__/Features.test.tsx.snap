// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@app/modules/main/screens/Home/components/Features renders successfully 1`] = `
<body>
  <div>
    <div>
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-lg-2"
        />
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-lg-8"
        >
          <div
            class="makeStyles-featuresContainer-1"
          >
            <h3
              class="makeStyles-featuresTitle-2"
            >
              Praesent sapien massa, convallis a pellentesque nec
            </h3>
          </div>
          <div
            class="makeStyles-featuresContent-3"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
                >
                  <div
                    class="MuiCardMedia-root makeStyles-media-4"
                    style="background-image: url(/static/images/intro1.jpg);"
                    title="Paella dish"
                  />
                  <div
                    class="MuiCardContent-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                    >
                      This impressive paella is a perfect party dish and a fun meal to cook together with your guests. Add 1 cup of frozen peas along with the mussels, if you like.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
                >
                  <div
                    class="MuiCardMedia-root makeStyles-media-4"
                    style="background-image: url(/static/images/intro2.jpg);"
                    title="Paella dish"
                  />
                  <div
                    class="MuiCardContent-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                    >
                      This impressive paella is a perfect party dish and a fun meal to cook together with your guests. Add 1 cup of frozen peas along with the mussels, if you like.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-lg-4"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
                >
                  <div
                    class="MuiCardMedia-root makeStyles-media-4"
                    style="background-image: url(/static/images/intro3.jpg);"
                    title="Paella dish"
                  />
                  <div
                    class="MuiCardContent-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                    >
                      This impressive paella is a perfect party dish and a fun meal to cook together with your guests. Add 1 cup of frozen peas along with the mussels, if you like.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-lg-2"
        />
      </div>
    </div>
  </div>
</body>
`;
